proxy/http_headers_test.go

```diff
@@ -495,5 +495,5 @@ func BenchmarkUint16Base16(b *testing.B) {
 			s = uint16base16(uint16(i))
 		}
 	})
-	_ = s == s // use the var to make go1.10 vet happy
+	fmt.Sprintf("hello %v", s) // use the var to make go1.10 vet happy
```

Comment:
Why is this preferable to the old impl?

Reply:
it was an attempt to avoid SA4000, but really just a non-functional change.

Reply:
Mm, fair enough. Why don't we use a `t.Logf` statement instead, then call it good.

Reply:
done.